sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -880,4 +880,13 @@ class SparkSqlParserSuite extends AnalysisTest with SharedSparkSession {
     parser.parsePlan("SELECT\u30001") // Unicode ideographic space
   }
   // scalastyle:on
+
+  test("Operator pipe SQL syntax") {
+    withSQLConf(SQLConf.OPERATOR_PIPE_SYNTAX_ENABLED.key -> "true") {
+      // Basic selection.
```

Comment:
hmm, shall we check the parsed plan?

Reply:
Sounds good, I added some checks. I don't want to compare all the parts of the plan because it's going to be a bit tedious and brittle, but I checked that the essential operators are there :)